### PR TITLE
addition of wild oats and wheat

### PR DESCRIPTION
**Pull request by Pr0man**
#### Summary
SUMMARY: Content "Adds new harvestable plants to fields and forests."

#### Purpose of change
I wish to add some wild grain-type plants which should be commonly found throughout New England because I felt that - while there are now more plants than what we used to have in earlier versions - some key plants I felt are relatively common were still missing.

#### Describe the solution
The additions I have made entail the following:
'wild wheat' and 'wild oat' plants, which are less nutritious (around 66% values) due to their small size and wild growth, not being properly domesticated plants, yet are semi-commonly found as "roadside flowers" basically all around the world, and according to some botany sources, also in many places around New England.
Also I added a somewhat generic "dead bush", which can sometimes be stumbled upon in fields, as well as common inside forests. Forests have many different plant materials which can be used as crude twine and the "withered plant" always felt like it was supposed to be just that: soft plant matter that can in a pinch be used for crude tools, but was always difficult to obtain when you consider how many plant materials inside a forest (shrubs, plant leaves etc.) could work for this matter and how many withered plants are asked for in many recipes. Either you had to deforest whole overmap tiles to create a makeshift rope by smacking bushes or hope for lucky finds in the foraging.
This bush could be seen as plants that due to different circumstances died off, but haven't decayed yet: think of trampling, blocked sunlight, insufficient nutrients. This gives the player a chance to specifically be on the hunt for withered plants when needed (Wilderness runs, early prep etc.), but one can simply chose to not collect them, making this an opt-in decision.
#### Describe alternatives you've considered
Alternatives I have considered would be to bring back the additional harvest entry for Underbrushes which always provided a certain amount of withered plants besides what you were collecting. I remember this being in the game for a short time, but got reverted after only a couple of versions - probably because people were complaining about the clutter the withered plants everywhere caused.

#### Additional context
When considering the addition of extra plants for the touch up on wilderness, I mainly consulted gobotany:

https://gobotany.newenglandwild.org/species/avena/fatua/

https://gobotany.newenglandwild.org/species/triticum/aestivum/

while also comparing nutritional values by judging size in comparision to their standard counterparts. These plants make up for their lack in "bulk" of nutrition with their sheer amount that you can find. Secluded fields behind houses that aren't tended to in particular are more often than not rifled with wild oatplants, which are considered to be a "weed", yet they can technically serve as nutritious food.

### CODE REVIEW

**Comment by AMurkin** [comment]
We have the [manual of style for descriptive text](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md). Please format the strings acording to it.

Use double sentence spacing after periods. This means that a period that ends a sentence should be followed by two spaces.
Use the serial comma (Oxford comma).

**Comment by Pr0man** [comment]
> 
> 
> We have the [manual of style for descriptive text](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md). Please format the strings acording to it.
> 
> Use double sentence spacing after periods. This means that a period that ends a sentence should be followed by two spaces.
> Use the serial comma (Oxford comma).

Addint in double spacing was no problem. After reading up on it I did however have the problem of not finding suitable locations for the serial comma so I decided to drop some commas which are not specifically required to potentially avoid confusion.

**Comment by Night-Pryanik** [comment]
Using commas isn't necessary. It means that IF you use commas, use Oxford comma rule.

File: data/json/furniture.json [hunk 1]
```diff
@@ -2314,6 +2314,62 @@
     "flags": [ "TRANSPARENT", "TINY", "FLAMMABLE_ASH", "NOCOLLIDE" ],
     "bash": { "str_min": 2, "str_max": 6, "sound": "crunch.", "sound_fail": "whish." }
   },
+  {
+    "type": "furniture",
+    "id": "f_wild_oats",
+    "name": "wild oats",
+    "description": "A bundel of wild oats which was oftentimes considered to be an unwanted weed overgrowing fields.  However that was before the cataclysm struck.",
```
Comment (matskuman5) [inline data/json/furniture.json]: bundel → bundle

File: data/json/items/comestibles/wheat.json [hunk 2]
```diff
@@ -205,6 +205,46 @@
...
+    "name": "wild oats",
+    "name_plural": "wild oats",
+    "weight": 66,
+    "color": "brown",
+    "comestible_type": "FOOD",
+    "symbol": ",",
+    "quench": -10,
+    "healthy": 1,
+    "calories": 211,
+    "description": "Wild oats, one of many roadside plants commonly found in New England.  Not as nutritious as it's domesticated counterpart yet still healthy and filling.",
```
Comment (matskuman5) [inline data/json/items/comestibles/wheat.json]: it's → its

File: data/json/items/comestibles/wheat.json [hunk 3]
```diff
@@ -205,6 +205,46 @@
...
+    "name": "wild wheat",
+    "name_plural": "wild wheat",
+    "weight": 176,
+    "color": "brown",
+    "comestible_type": "FOOD",
+    "symbol": ",",
+    "quench": -10,
+    "healthy": 1,
+    "calories": 394,
+    "description": "Wild wheat which can be commonly found in the fields of New England.  In comparision to normal wheat, the oars and resulting grains are smaller in size.",
```
Comment (matskuman5) [inline data/json/items/comestibles/wheat.json]: comparision → comparison

**Comment by Pr0man** [comment]
> 
> 
> typos

Many thanks on the proofreading and for me to take notes: Do not work on a PR if you haven't slept/just woke up and are not a native English speaker.

**Comment by Pr0man** [comment]
Accidentally closed the PR and messed everything up on my end, yikes.

**Pull request closed without merging**